xbout/tests/test_load.py

```diff
@@ -353,3 +353,23 @@ def test_trim_ghosts(self):
         selection = {'time': slice(2, -2)}
         expected = ds.isel(**selection)
         xrt.assert_equal(expected, actual)
+
+    def test_trim_timing_info(self):
+        ds = create_test_data(0)
+        from xbout.load import _bout_timing_variables
+
+        # remove a couple of entries from _bout_timing_variables so we test that _trim
+        # does not fail if not all of them are present
+        _bout_timing_variables = _bout_timing_variables[:-2]
+
+        for v in _bout_timing_variables:
+            ds[v] = 42.
+        expected = create_test_data(0)
+        try:
+            xrt.assert_equal(ds, expected)
+        except AssertionError:
```

Comment:
I see what you're trying to do here, but I don't think it adds much compared to just using the final `xrt.assert_equal(ds, expected)` assertion. This first assertion will pass if any assertion error is raised at all, even the wrong one, and asserting False won't give you details about what's in the dataset on failure like `assert_equal` does. I would just only keep the final assertion.